.github/workflows/testing.yml

```
@@ -15,7 +15,12 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - run: pip install numpy
-      - run: pip install pytest
-      - run: pip install -U coverage
+      - run: pip install -U pytest coverage coveralls PyYAML
       - run: coverage run -m --source=. pytest
+      - run: coveralls --output=coverage.json
+      - run: coveralls-lcov -v -n coverage.info > coverage.json
```

I'm not sure how this works.... (or if it works?) There is no new update on Coveralls associated with this commit.

I think you can replace this with:
```suggestion
      - run: coverage lcov
```

This should generate an LCOV file that the action can use.